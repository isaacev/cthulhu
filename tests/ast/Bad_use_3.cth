use ::Foobar;

#[entry]
fn main() -> () {
  -- do nothing
}
