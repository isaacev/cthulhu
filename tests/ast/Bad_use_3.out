
  UNKNOWN LIBRARY  -----------------------------------------------------------

    : Bad_use_3
  1 | use ::Foobar;
    |       ^^^^^^

  The program referenced a library that could not be found in the following
  places:

    /Users/isaac/code/php/cthulhu/src/stdlib

  These were the libraries that look the most similar:

    Fmt
    Io
    Kernel

